tests/models/test_dag.py

```diff
@@ -3429,8 +3429,8 @@ def consumer(value):
         state=TaskInstanceState.SUCCESS,
         session=session,
     )
-    assert dr1 in session, "Check session is passed down all the way"
-
+    dag_runs = session.query(DagRun).filter(DagRun.run_id == dr1.run_id).all()
+    assert len(dag_runs) > 0, "Check session is passed down all the way"
```

Comment:
We can use `func.count()` here to improve performance. This doesn’t matter that much since this is just a test (and if `all()` returns anything to cause performance issues, we should fix it anyway) but that’s good practice.